scripts/onnx_generator/__main__.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import os
+import sys
+import re
+from .OperatorHeader import OperatorHeader
```

Comment:
I'm having issues both in the terminal and pycharm with the leading `.`. Can we just remove it?

Reply:
I can't execute it without it. did you run it as a model (`python -m onnx_generator`)?